lib/presentation/models/word_cloud/word_cloud_data.dart

```diff
@@ -0,0 +1,78 @@
+import 'package:flutter/painting.dart';
+import 'dart:ui';
+
+class FlutterHashtag {
+  const FlutterHashtag(
+    this.hashtag,
+    this.color,
+    this.size,
+    this.rotated,
+  );
+  final String hashtag;
+  final Color color;
+  final int size;
+  final bool rotated;
```

Comment:
Can you rename it to `isRotated`

Reply:
Addressed in commit 6a2865e61273f2a55d1cc7a4c0f773e8ab323c49